proxysql-admin

```diff
@@ -267,6 +267,8 @@ One of the options below must be provided.
   --enable, -e                       Auto-configure Percona XtraDB Cluster nodes into ProxySQL
   --update-cluster                   Updates the cluster membership, adds new cluster nodes
                                      to the configuration.
+  --update-mysql-version             Updates mysql server version in proxysql db based on
```

Comment:
May want to say something more like
Updates the mysql-server_version variable in ProxySQL with the version from a node in the cluster.

Reply:
Changed text.